src/pages/FilterMenu.css

```diff
@@ -117,18 +117,23 @@ ul.FilterMenu-trendy li.FilterMenu-trendy-title {
   color: rgba(255, 255, 255, 0.5);
 }
 
-.FilterMenu-form input.FilterMenu-valid {
-  display: none;
-  border: 0px;
-  width: 70px;
-  height: 70px;
-  font-size: 3em;
-  line-height: 1.5em;
-  text-align: center;
-  color: #FFFFFF;
-  border-radius: 35px;
-  background: url(../img/btn-check.png) no-repeat center center;
-  order: 0;
-  flex: 0 1 auto;
-  align-self: center;
+.FilterMenu-checkButton {
+width: 70px;
+height: 70px;
+border-radius: 40px;
```

Comment:
If you want to have it be 100% round the border-radius needs to be half the width and height. So 35px in this case